src/fontlibc/fontlibc.h

```diff
@@ -0,0 +1,437 @@
+/**
+ * @file
+ * @authors DrDnar
+ * @brief Provides improved font support
+ */
+
+#ifndef H_FONTLIBC
+#define H_FONTLIBC
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define FONTLIB_ENABLE_AUTO_WRAP 1
+#define FONTLIB_AUTO_CLEAR_TO_EOL 2
+#define FONTLIB_PRECLEAR_NEWLINE 4
+
+typedef enum {
+	/* clear = sans-serif font */
+	SERIF = 0x01,
+	/* If both are set, then assume there's no difference between oblique
+	 * and italic styles. */
+	OBLIQUE = 0x02,
+	ITALIC = 0x04,
+	/* Chances are you're not using this library for monospaced fonts.
+	 * But if you are, you'll still have to provide a widths table where
+	 * every byte is the same. */
+	MONOSPACED = 0x08
+} font_styles_t;
```

Comment:
```suggestion
} fontlib_styles_t;
```

Following the pattern of other libraries (which is good for scoping), all definitions (not just functions) should share a name prefix.

Reply:
Fixing